lib/dockerize_rails/helpers.rb

```diff
@@ -20,6 +20,19 @@ def self.ensure_rails_root
       0
     end
 
+    def self.params_help(command)
+      if command.key? :params
+        [command[:params].keys.map do |param|
+          "\n          #{('[' + param.to_s + ']').ljust(15, ' ')} -- #{command[:params][param]}"
+        end,
+         "\n"].join
+      else
+        ''
+      end
+    end
+
+    # rubocop:disable Metrics/MethodLength
+    # rubocop:disable Metrics/AbcSize
```

Comment:
__[rubocop]__
* Unnecessary disabling of `Metrics/AbcSize`.

<!-- Comment by [SideCI](https://sideci.com) -->